File: Iniciante/Python/1013.O Maior.py
a, b, c = input().split()

a = int(a)
b = int(b)
c = int(c)

maiorAB = (a+b+abs(a-b))/2

maiorAC = (maiorAB+c+abs(maiorAB-c))/2
maiorAC = int(maiorAC)

print(maiorAC,'eh o maior')

File: Iniciante/C++/1013.O maior.cpp
#include <iostream>
#include <stdlib.h>

using namespace std;

int main(void){
    int a, b, c, maiorAB, maiorAC;

    cin >> a >> b >> c;

    maiorAB = (a+b+abs(a-b))/2;
    maiorAC = (maiorAB+c+abs(maiorAB-c))/2;

    cout << maiorAC << " eh o maior" << endl;


    return 0;
}